packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -435,6 +451,10 @@ class MessageList extends Widget<Properties> {
 
     switch (name) {
       case 'currentUserId':
+      case 'showDayHeaders':
```

Comment:
[copypaste]
We have copypaste now, it's duplicated below. Kindly union all cases when `_invalidate` is called

Reply:
e6f6146790b208b2332e9d492319db37dd726ad1